Introduction to Coroutines and Channels/06_StructuredConcurrency.md

```diff
@@ -32,39 +32,39 @@ fun main() = runBlocking { /* this: CoroutineScope */
 }
 ```
 
-When we call `launch` inside `runBlocking`, we call it as an extension to the implicit receiver
+If we were to call `launch` inside `runBlocking`, we would call it as an extension to the implicit receiver
 of the `CoroutineScope` type.
 Alternatively, we could explicitly write `this.launch`.
 
-We can say that the nested coroutine (started by `launch` in this example) is a child of the outer coroutine
+We could say that the nested coroutine (started by `launch` in this example) is a child of the outer coroutine
 (started by `runBlocking`). This "parent-child" relationship works through scopes: the child coroutine is started from
 the scope corresponding to the parent coroutine.
 
 It is possible to create a new scope without starting a new coroutine.
 The `coroutineScope` function does this. 
-When you need to start new coroutines in a structured way inside a `suspend` function without access to the outer scope,
+If we ever needed to start new coroutines in a structured way inside a `suspend` function without access to the outer scope,
 for example inside `loadContributorsConcurrent`,
-you can create a new coroutine scope which automatically becomes a child of the outer scope that this `suspend` function
+we could create a new coroutine scope which automatically becomes a child of the outer scope that this `suspend` function
 is called from.
 
 It's also possible to start a new coroutine from the global scope using `GlobalScope.async` or `GlobalScope.launch`.
 This will create a top-level "independent" coroutine.
 
 The mechanism providing the structure of the coroutines is called "structured concurrency".
-Let's see what benefits the structured concurrency has over global scopes:
+Let's see what benefits structured concurrency has over global scopes:
 
 * The scope is generally responsible for child coroutines,
 and their lifetime is attached to the lifetime of the scope. 
 * The scope can automatically cancel child coroutines if something goes wrong or
-if the user simply changes their mind and decides to revoke the operation.
+if a user simply changes their mind and decides to revoke the operation.
 * The scope automatically waits for completion of all the child coroutines.
 Therefore, if the scope corresponds to a coroutine, then the parent coroutine does not complete until all the coroutines
 launched in its scope are complete.
 
 When using `GlobalScope.async` there is no structure that binds several coroutines to a smaller scope.
-The coroutines started from the global scope are all independent; 
+The coroutines started from a global scope are all independent; 
```

Comment:
If there's one global scope (`GlobalScope`), shouldn't we say the global scope here?